tests/func/test_data_cloud.py

```diff
@@ -149,6 +152,21 @@ def get_ssh_url():
     )
 
 
+def get_ssh_url_mocked(user, port):
+    if os.name == "nt":
+        # Since we use posixpath to compose paths.
+        # With self.prefix = C:\some\path we'll get something
+        # like C:\some\path\/new/path, which is not right.
+        # So, we need to make the path consistence
+        path = get_local_storagepath().replace("\\", "/")
+        url = "ssh://{}@127.0.0.1:{}:{}".format(user, port, path)
```

Comment:
Continuing from https://github.com/iterative/dvc/pull/1911#discussion_r278794297 .

> No, It will raise an exception: DvcException: couldn't get the '/c:/users/cao/appdata/local/temp/dvc-test.7968.outdjx.e8373c98-b117-479c-9ed3-000a974182da' remote directory files list . Because the initial slash / before c:/. That is the reason I provide this indicator to solve in Windows. Is that make sense or do you have other solution?

But you are not passing the leading slash at all right now. What you are currenty using as an url is `ssh://example.com:88:c:/path/to`, which when parsed by urlparse results in:
```
>>> urlparse.urlparse("ssh://example.com:88:c:/path/to")
ParseResult(scheme='ssh', netloc='example.com:88:c:', path='/path/to', params='', query='', fragment='')
```
so your prefix is `/path/to` and not `c:/path/to`. I'm wondering if it is even accessing the proper directory.

And if you remove `:` as I'm asking, you'll be passing `ssh://example.com:88c:/path/to`, which will result in
```
>>> urlparse.urlparse("ssh://example.com:88c:/path/to")
ParseResult(scheme='ssh', netloc='example.com:88c:', path='/path/to', params='', query='', fragment='')
```
which is totally wrong.

Reply:
The reason why it worked is probably because paramiko server is running in `C:\`, so it becomes its root and `/path/to` is resolved relative to it. Which makes sense and is indeed in sync with how ssh servers work on windows. So we should use not `c:/path/to`, but rather `/path/to` in our urls(e.g. `ssh://example.com:22:/path/to` or `ssh://example.com:22/path/to`).

Reply:
Hm. I don't quite understand what you want me to change about this point. Could you elarboate more? Thank you

Reply:
Ah, I got your point now

Reply:
Your ssh server has root set to `C:\\`, so it will resolve paths relative to that. Which means that you need to use `/path/to` as `path` in the `url`.

Reply:
DONE, sir :)